fontbe/src/orchestration.rs

```diff
@@ -210,12 +213,26 @@ where
     T::read(FontData::new(&buf)).unwrap()
 }
 
-fn raw_hmtx_from_file(file: &Path) -> RawHmtx {
+pub struct Bytes {
+    buf: Vec<u8>,
+}
+
+impl Bytes {
+    pub(crate) fn new(buf: Vec<u8>) -> Bytes {
+        Bytes { buf }
+    }
+
+    pub fn get(&self) -> &[u8] {
+        &self.buf
+    }
+}
+
+fn raw_from_file(file: &Path) -> Bytes {
```

Comment:
does this want to be `Bytes::read_from_file` or something?

Reply:
That seems to complicate my macro. Will add a comment.